modules/kmm-running-depmod.adoc

```diff
@@ -16,7 +16,8 @@ You must have a Red Hat subscription to download the `kernel-devel` package.
 
 .Procedure
 
-. To generate `modules.dep` and `.map` files for a specific kernel version, run `+depmod -b /opt ${KERNEL_VERSION}+`.
+. To generate `modules.dep` and `.map` files for a specific kernel version, run
+`+depmod -b /opt ${KERNEL_VERSION}+`.
```

Comment:
Also even if you didn't write this line, this should honestly be rewritten to better resemble our typical styling:
```suggestion
. Generate `modules.dep` and `.map` files for a specific kernel version by running the following command:
+
[source,terminal]
----
$ depmod -b /opt <kernel_version>`.
----
```

(I'm assuming the kernel version is an input value for the user and not a variable that's already set, if that's wrong then I think `${KERNEL_VERSION}` can be left as is)

Reply:
Changed. Also, the variable is already set, so leaving in **${KERNEL_VERSION}+**.